src/ol_infrastructure/applications/codejail/Pulumi.applications.codejail.mitx.Production.yaml

```diff
@@ -4,8 +4,8 @@ encryptedkey: AQICAHg/+QzF9hGIaoayDitgnEVHEhuaANONVQQOnqpkIsol1gFmLVxyy2hwwSaJxQ
 config:
   aws:region: us-east-1
   codejail:auto_scale:
-    desired: 5
-    max: 10
+    desired: 2
```

Comment:
We should probably keep this at 3 for a minimum. The steady state usage is low, but we periodically get large peaks when classes run an exam that require a codejail.